### ISSUE TITLE
[KC17] [Docs] Running Keycloak in a container: feature names ?

### ISSUE THREAD

**Issue by semangard** [keycloak/keycloak#10861]
### Describe the bug

Hello,

The doc : https://www.keycloak.org/server/containers
indicates
that features can be activated with for ex: ENV KC_FEATURES=token-exchange

Could you please check the name of the feature : is the '-' correct ? shouldn't it be '_'  like indicating by the doc https://www.keycloak.org/docs/latest/server_installation/#profiles
![image](https://user-images.githubusercontent.com/57065741/159446727-fd7d1aac-ddd3-476c-83ad-d28c2505e3b3.png)

Could you please 
- either fix the doc
- or add a comment indicating that feature's name should be renammed : '-' replaced by '_' 

### Version

17.0.0

### Expected behavior

Doc https://www.keycloak.org/server/containers more accurate or correct

**Comment by DGuhr** [comment]
Hey @semangard thanks for noticing the inconsistency. Actually for quarkus it is `token-exchange`, not `token_exchange`, as we tried to use one scheme for all (dashcase). 

Also, when you open `http://www.keycloak.org/documentation.html` and click on the server installation guide for 17.0.0, you're getting to the new guides we created for the quarkus distribution (standard since 17.0.0). So basically the link you posted is for the legacy wildfly distribution, so the "17.0.0" for it means actually "17.0.0-legacy", the 17 version of wildfly. 

For quarkus there's the container guide you already mentioned, and also https://www.keycloak.org/server/features this guide showing the right feature with the right dash-cased name.

@stianst Is it possible to change the title of the server installation guide for the legacy dist to include "legacy" or sth around these lines? At the moment you get it as you see in the screenshot, which might be fine when coming from the documentation page and click on the link under category "Deprecated WildFly distribution" but not when you open the guide directly, then its ambiguous if it is quarkus or wildfly and I guess this issue arose from this confusion.

<img width="1356" alt="Bildschirmfoto 2022-03-22 um 12 43 26" src="https://user-images.githubusercontent.com/89905860/159474569-ac48531a-8aa4-4b04-afd8-e44ed967f478.png">
